Backpack-SwiftUI/Radii/Classes/Generated/BPKRadii.swift

```diff
@@ -0,0 +1,58 @@
+// clang-format off
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKCornerRadius {
+    public let value: CGFloat
+    
+    private init(value: CGFloat) {
+        self.value = value
+    }
+}
+
+public extension RoundedRectangle {
+    init(cornerRadius: BPKCornerRadius, style: RoundedCornerStyle = .circular) {
+        self.init(cornerRadius: cornerRadius.value, style: style)
+    }
+}
+
+public extension View {
+    func cornerRadius(_ cornerRadius: BPKCornerRadius, style: FillStyle = FillStyle()) -> some View {
+        clipShape(RoundedRectangle(cornerRadius: cornerRadius), style: style)
+    }
+}
```

Comment:
Would it be possible to extend the RoundedRectangle clipshape to accept the BPKCornerRadius? 

This way we stay 'closer' to the original and allow ourselves to introduce other clipshapes that also support the corner radii.

Reply:
hey, I didn't quite get this, I did extend RoundedRectangle to make it receive a BPKCornerRadius, is that what you mean?